package.json

```diff
@@ -37,7 +37,7 @@
     "connected-react-router": "6.4.0",
     "copy-to-clipboard": "3.2.0",
     "deep-diff": "1.0.2",
-    "giantswarm-v4": "git+https://github.com/giantswarm/giantswarm-js-client.git#master",
+    "giantswarm-v4": "git+https://github.com/giantswarm/giantswarm-js-client.git#app-config",
```

Comment:
TODO: change this back to master when api-spec and giantswarm-js-client are settled.